kgforge/specializations/stores/bluebrain_nexus.py

```diff
@@ -627,11 +627,6 @@ def search(
             **params
     ) -> List[Resource]:
 
-        if "filters" in params:
```

Comment:
is this specific to ES views?

Reply:
specifying filters through search used to be done with *filters, and in a previous MR, it was changed to filters. I changed it back to *filters but I forgot to change to the forge call to the store

Reply:
I see, yes, I remember the `*filters` argument.